.gitignore

```diff
@@ -188,6 +188,10 @@ samples/client/petstore/kotlin-string/build
 samples/server/petstore/kotlin-server/ktor/build
 \?
 
+# Kotlin Spring Boot
+samples/server/petstore/kotlin-springboot
+samples/server/openapi3/petstore/kotlin-springboot
+
```

Comment:
@dr4ke616 Can you please remove these lines so that the project will include these files to be tested by the CI?

Reply:
I was going to ask you about this, then I saw that you commented on it.

Putting these directories into .gitignore is a great way to reduce noise while doing local development and awaiting feedback, though!